BangazonWorkForceManagement/Models/ViewModels/ComputerDeleteViewModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonWorkForceManagement.Models.ViewModels
+{
+    public class ComputerDeleteViewModel
+    {
+        public int Id { get; set; }
+
+
+        public DateTime PurchaseDate { get; set; }
+
+
+        public string Make { get; set; }
+
+
+        public string Manufacturer { get; set; }
+
+        public bool DisplayDelete { get; set; }
```

Comment:
It's best practice to use a verb phrase for boolean values, (e.g. `ShouldDisplayDelete`)